pkgs/applications/editors/vim/plugins/generated.nix

```
@@ -9930,6 +9930,18 @@ final: prev:
     meta.homepage = "https://github.com/Almo7aya/openingh.nvim/";
   };
 
+  one-small-step-for-vimkind = buildVimPlugin {
+    pname = "one-small-step-for-vimkind";
+    version = "2024-11-07";
```

```suggestion
    version = "2024-10-15";
```
https://github.com/jbyuki/one-small-step-for-vimkind/commit/ad065ad2c814249cfb9e344ce5b2b35d36fbc09f